bin/brew

```diff
@@ -44,4 +44,17 @@ fi
 
 HOMEBREW_LIBRARY="$HOMEBREW_REPOSITORY/Library"
 
-source "$HOMEBREW_LIBRARY/Homebrew/brew.sh"
+if [[ -n "$HOMEBREW_ENV_FILTERING" ]]
+then
+  PATH="/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
> Persist original PATH in ORIG_PATH and optionally restore it later?

Yeh, this. Won't necessarily need this for this PR but that seems like the right direction to take.